bundles/org.openhab.binding.vdr/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="vdr" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="vdr">
+		<label>VDR - The Video Disk Recorder</label>
+		<description>VDR - The Video Disk Recorder (https://tvdr.de)</description>
+
+		<channels>
+			<channel id="power" typeId="vdrPower"/>
+			<channel id="channel" typeId="vdrChannel"/>
+			<channel id="channelName" typeId="vdrChannelName"/>
+			<channel id="volume" typeId="vdrVolume"/>
+			<channel id="recording" typeId="vdrRecordingActive"/>
+			<channel id="diskUsage" typeId="vdrDiskUsage"/>
+			<channel id="message" typeId="vdrMessage"/>
+			<channel id="keyCode" typeId="vdrKeyCode"/>
+			<channel id="currentEventTitle" typeId="vdrEventTitle">
+				<label>Current EPG Event Title</label>
+			</channel>
+			<channel id="currentEventSubTitle" typeId="vdrEventSubTitle">
+				<label>Current EPG Event Sub Title</label>
+			</channel>
+			<channel id="currentEventBegin" typeId="vdrEventBegin">
+				<label>Current EPG Event Begin</label>
+			</channel>
+			<channel id="currentEventEnd" typeId="vdrEventEnd">
+				<label>Current EPG Event End</label>
+			</channel>
+			<channel id="currentEventDuration" typeId="vdrEventDuration">
+				<label>Current EPG Event Duration in Minutes</label>
+			</channel>
+			<channel id="nextEventTitle" typeId="vdrEventTitle">
+				<label>Next EPG Event Title</label>
+			</channel>
+			<channel id="nextEventSubTitle" typeId="vdrEventSubTitle">
+				<label>Next EPG Event Sub Title</label>
+			</channel>
+			<channel id="nextEventBegin" typeId="vdrEventBegin">
+				<label>Next EPG Event Begin</label>
+			</channel>
+			<channel id="nextEventEnd" typeId="vdrEventEnd">
+				<label>Next EPG Event Begin</label>
+			</channel>
+			<channel id="nextEventDuration" typeId="vdrEventDuration">
+				<label>Next EPG Event Duration in Minutes</label>
+			</channel>
+		</channels>
+
+		<properties>
+			<property name="version">VDR Version</property>
+		</properties>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Hostname / IP Address</label>
+				<description>Hostname or IP Address of VDR instance</description>
```

Comment:
You could specify `<context>network-address</context>` to get a free validation.